### PR TITLE
operator: Replace dead documentation link

### LINKED ISSUE #24143: Operator docs link is broken
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

operator

### Describe the bug

The documentation link for the operator points to a 404: https://www.keycloak.org/docs/latest/server_installation/index.html#_operator

### Version

22.0.4

### Expected behavior

The docs link points to a docs page

### Actual behavior

The docs link points to a non-existing page

### How to Reproduce?

Go to the [Operator Hub page for Keycloak](https://operatorhub.io/operator/keycloak-operator) (any version) and click on "Documentation" on the right-hand side bar

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pierreprinetti**
Before this patch, the documentation link generated for the Operator page resulted in a 404 Not Found on Github pages.

With this change, the documentation link points to the "Operator" anchor in the Keycloak docs index. Note that the operator docs don't seem to be versioned on the Keycloak website.

Fixes #24143

### CODE REVIEW

**Comment by pierreprinetti** [comment]
Rebased.

**Comment by pierreprinetti** [comment]
Rebased.

**Comment by pierreprinetti** [comment]
hmm, maybe I should ping someone. @jonkoops?

**Review by jonkoops — APPROVED** [review]
I am not a maintainer so I cannot merge this, but changes look good to me.

**Comment by pierreprinetti** [comment]
@vmuzikar Is there anything else expected from me, before this PR can be merged?

**Comment by jonkoops** [comment]
@pierreprinetti Please do not @ other team members, we're already pretty busy and this just adds more notifications for us. Your PR will be picked up when the maintainers have time to do so.

**Comment by jonkoops** [comment]
Side-bar. Could you perhaps log an issue for this fix? This is something we might want to backport to the existing release, and we need an issue to track that.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@pierreprinetti Thanks for the PR!

Change LGTM but we need the GH Issue as mentioned by @jonkoops. Please see https://github.com/keycloak/keycloak/blob/main/CONTRIBUTING.md

**Comment by jonkoops** [comment]
@vmuzikar issue us now linked, can you take another look?

**Review by vmuzikar — APPROVED** [review]

**Pull request merged**
